DeUrgenta/Features/EmergencyBackpack/Model/InviteFriendsContextType.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+enum InviteFriendsConextType {
```

Comment:
@TiboldiRoberto Is this enum used anymore?

Reply:
No :)